app/models/manageiq/providers/ansible_tower/shared/provider.rb

```diff
@@ -58,8 +58,69 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def create_sandbox_data
+    with_provider_connection do |c|
+      organization = ensure_organization(c)
+      inventory = ensure_inventory(c, organization.id)
+      ensure_host(c, inventory.id)
+    end
+  end
+
+  def default_organization(connection = connect)
+    @@default_org ||= connection.api.organizations.all(
```

Comment:
Well, I don't want it to be an instance variable because I want to cache more aggressively than that. I don't have much experience with class instance variables, but it feels like I would need to write explicit accessors to use them from an instance context, right? Plus it's all a bit more complicated because this is a mixin ...

I'm not crazy about any of the options, what do you think?